src/modules/users/models/user.model.ts

```diff
@@ -1,4 +1,5 @@
 import { Field, ID, ObjectType } from '@nestjs/graphql';
+import { Role } from './role.model';
 
 @ObjectType({ description: 'User' })
```

Comment:
`Description` is redundant, either add full description or omit it.

Reply:
can't be omitted, it will throw an error. @falyoun